Sources/pointfreeco/Transcripts/AllEpisodes.swift

```diff
@@ -0,0 +1,7 @@
+import PointFree
+
+func allEpisodes() -> [Episode] {
```

Comment:
This is a stub that returns a stubbed episode for OSS. It gets overwritten by the private repo.